clients/java/src/main/java/io/zeebe/client/impl/oauth/OAuthCredentialsProviderBuilder.java

```diff
@@ -163,8 +216,29 @@ private void validate() {
         throw new IllegalArgumentException(
             "Expected specified credentials cache to be a file but found directory instead.");
       }
+      if (connectTimeout.isZero() || connectTimeout.isNegative()) {
+        throw new IllegalArgumentException("Expected connectTimeout to be is more that 0.");
+      }
```

Comment:
```suggestion
```
This check is duplicated and can be removed.

Reply:
Done